Automattic/wp-calypso
# Added React.forwardRef to FormLabel component

On  `DeleteUser` component a `ref` is added to the `FormLabel` component.

```
...
   <FormFieldset>
      <FormLabel ref={ this.setReassignLabel }>
         <FormRadio
...
```

But `FormLabel` is a function component, so `ref`s don't work out of the box.

![image](https://user-images.githubusercontent.com/3801502/139489770-1d5e6bc3-d8c5-4463-8aa7-71ff0ea3cc91.png)

This PR uses `React.forwardRef()` to forward the ref to `FormLabel`'s child DOM element, a `<label />`

By doing this, the error disappears and, in this specific use case, the intended behavior is set: when clicking the label, the popover doesn't close.

![image](https://user-images.githubusercontent.com/3801502/139490995-0388e11c-e001-4079-8702-0c5296263776.png)


#### Testing instructions

- Open Calypso on any site.
- Go to Users
- Click on a member from the list
- Check the browser log
